Documentation/ApiOverview/SoftReferences/Index.rst

```diff
@@ -154,6 +171,42 @@ The default set up is found in :file:`typo3/sysext/core/Configuration/DefaultCon
         // ...
     ],
 
+Examples
+========
+
+For the `tt_content.bodytext` field of type text from the example
+above, the configuration looks like this::
+
+   $GLOBALS['TCA']['tt_content']['columns']['bodytext'] =>
+      // ...
+
+      'config' => [
+         'type' => 'text',
+         'softref' => 'typolink_tag,email[subst],url',
+         // ...
+      ],
+
+      // ...
+   ];
+
+This means, the parsers for the softref types `typolink_tag`, `email` and
+`url` will all be applied. The email soft reference parser gets the additional
+parameter 'subst'.
+
+The content could look like this:
+
+.. code-block:: html
+
+   <p><a href="t3://page?uid=96">Congratulations</a></p>
+   <p>To read more about <a href="http://example.org/some-cool-feature">this cool feature</a></p>
+   <p>Contact: email@example.org</p>
+
+The parsers will return an array containing information about
+
+#. A page link
+#. An external link
+#. An email link
```

Comment:
Maybe add the specific values of this example for clarification?

Reply:
Made a more specific example